include/rocksdb/listener.h

```diff
@@ -180,7 +179,8 @@ class EventListener {
   // @param ci a reference to a CompactionJobInfo struct. 'ci' is released
   //  after this function is returned, and must be copied if it is needed
   //  outside of this function.
-  virtual void OnCompactionCompleted(DB *db, const CompactionJobInfo& ci) {}
+  virtual void OnCompactionCompleted(DB* /*db*/,
+      const CompactionJobInfo& /*ci*/) {}
```

Comment:
IMO this non-uniformity is confusing. Are you implying that non-language standard types don't need comments, as they're self-explanatory?